client/src/ui-kit/toast.js

```diff
@@ -0,0 +1,81 @@
+import styled, { css } from "styled-components";
```

Comment:
👏 this rules! 
your indentation is wacky though, do you have prettier installed? it will clean it right up for you